### ISSUE TITLE
poliastro.twobody.Orbit import error

### ISSUE THREAD

**Issue by ReeceHumphreys** [poliastro/poliastro#1493]
Importing Orbit from poliastro.twobody results in an attribute error.

🐞 **Problem**

On macOS 12.3, I performed an install using pip and the instructions for downloading the latest build from GitHub and am getting an attribute error when importing  `Orbit` as follows:

```
from poliastro.twobody import Orbit
```

Here is the resulting Attribute error that got output:
```
AttributeError                            Traceback (most recent call last)
/Users/reecehumphreys/Developer/Personal/ODAP/notebooks/Introduction.ipynb Cell 27' in <cell line: [1]()>()
----> 1[ from poliastro.twobody import Orbit

File ~/.virtualenvs/odap/lib/python3.9/site-packages/poliastro/twobody/__init__.py:1, in <module>
----> ]()[1]()[ from poliastro.twobody.orbit import Orbit
      ]()[3]()[ __all__ = ["Orbit"]

File ~/.virtualenvs/odap/lib/python3.9/site-packages/poliastro/twobody/orbit/__init__.py:1, in <module>
----> ]()[1]()[ from poliastro.twobody.orbit.scalar import Orbit
      ]()[3]()[ __all__ = ["Orbit"]

File ~/.virtualenvs/odap/lib/python3.9/site-packages/poliastro/twobody/orbit/scalar.py:6, in <module>
      ]()[4]()[ import numpy as np
      ]()[5]()[ from astropy import time, units as u
----> ]()[6]()[ from astropy.coordinates import (
      ]()[7]()[     ICRS,
      ]()[8]()[     CartesianDifferential,
      ]()[9]()[     CartesianRepresentation,
     ]()[10]()[     get_body_barycentric,
     ]()[11]()[ )
     ]()[13]()[ from poliastro.frames.util import get_frame
     ]()[14]()[ from poliastro.threebody.soi import laplace_radius

File ~/.virtualenvs/odap/lib/python3.9/site-packages/astropy/coordinates/__init__.py:16, in <module>
     ]()[14]()[ from .earth import *
     ]()[15]()[ from .transformations import *
---> ]()[16]()[ from .builtin_frames import *
     ]()[17]()[ from .name_resolve import *
     ]()[18]()[ from .matching import *

File ~/.virtualenvs/odap/lib/python3.9/site-packages/astropy/coordinates/builtin_frames/__init__.py:32, in <module>
     ]()[30]()[ from .fk4 import FK4, FK4NoETerms
     ]()[31]()[ from .galactic import Galactic
---> ]()[32]()[ from .galactocentric import Galactocentric, galactocentric_frame_defaults
     ]()[33]()[ from .supergalactic import Supergalactic
     ]()[34]()[ from .altaz import AltAz

File ~/.virtualenvs/odap/lib/python3.9/site-packages/astropy/coordinates/builtin_frames/galactocentric.py:591, in <module>
    ]()[587]()[     return get_matrix_vectors(galactocentric_coord, inverse=True)
    ]()[590]()[ # Create loopback transformation
--> ]()[591]()[ frame_transform_graph._add_merged_transform(Galactocentric, ICRS, Galactocentric)

AttributeError: 'TransformGraph' object has no attribute '_add_merged_transform']()
```

This seems to be an issue with AstroPy looking at the stack trace but I figured that I would bring it to your attention!

**Comment by astrojuanlu** [comment]
Hello @ReeceHumphreys , thanks for opening this issue! Indeed, it looks like we're incompatible with the latest Astropy. Could you please try `pip install "astropy<5"` and try again?

**Comment by ReeceHumphreys** [comment]
Hello @astrojuanlu , that fixed the issue!

**Comment by astrojuanlu** [comment]
Thanks for checking! We'll need to do a bugfix release, it will happen in a few months.

**Comment by yakkonaut** [comment]
Linking https://github.com/poliastro/poliastro/issues/1496 here. Going to start taking a look at this. Will post to the matrix server too- I don't want to duplicate anyone's effort but happy to take a swing at the update here.

**Comment by kerel-fs** [comment]
Thanks @yakkonaut for cross-linking this issue! I failed to find it during my previous search when investigating the version pin for astropy.


---


With a local checkout of branch `0.16.x` I can't reproduce the import issue for `poliastro.twobody.Orbit` stated here.

**Steps performed**:
- Checkout branch `0.16.x`
- Modify version pinning in `pyproject.toml` to `astropy>=5`
- Install via `pip install -e .`
-  Run this code: `from poliastro.twobody import Orbit` --> no import error

Running `tox -e py310-fast` succeeds as well, see [tox_py310-fast.log](https://github.com/poliastro/poliastro/files/8492507/tox_py310-fast.log).

**Comment by astrojuanlu** [comment]
Thanks all for checking this. Perhaps this was a temporary problem that got fixed in a subsequent Astropy release, but I can't find any references to it in the Astropy tracker. If it disappeared, we can close and just remove the pinning in 0.16.1.

**Comment by astrojuanlu** [comment]
Closed in poliastro 0.16.3 🚀
